src/components/Personal-Zone/Publisher/JobsAppEmployerBody.tsx

```diff
@@ -31,11 +32,14 @@ const JobsAppEmployerBody: React.FC<JobsAppEmployerBodyOwnProps> = (props): JSX.
         jobsStore.loadAdvertisements();
     }, [jobsStore]);
 
+    const showEmptyDisplay = false && jobsStore.advertisements.length > 0; 

```

Comment:
`false` is not necessary here ^^, as it always depend on the 2nd clause

Reply:
You are confusing between the 2 screens:
1. @SteveVaknin did this for `Ramad`
2. You are talking about `Candidate screen`, where the candidate can `upload a resume`